bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -328,6 +339,22 @@
             return registry;
         }
 
+        function getPointer(obj) {
+            if (typeof obj === 'string') {
```

Comment:
It would be good to drop the block scope for single-statement bodies for consistency.